include/renderer_gl/opengl.hpp

```diff
@@ -615,83 +615,4 @@ namespace OpenGL {
         glBlendFuncSeparate(fac1, fac2, fac3, fac4);
     }
 
-    // Abstraction for GLSL vectors
-    template <typename T, int size>
```

Comment:
There was no way for you to know this lol but I'll bring these back. Reason is that this header is actually a standalone library several other projects use so I need to be able to drag and drop the latest Panda version to those projects. This means I can't make breaking ABI changes like this

Reply:
The header should be in externals then

Reply:
True imma move it when I'm on the keeb again